app/assets/stylesheets/admin/styles.scss

```diff
@@ -66,6 +66,10 @@ form {
   width: 60px;
 }
 
+.block.checkbox label {
```

Comment:
.block is non-semantic div name... .form-actions or something more human readable..